archinstall/locales/pl/LC_MESSAGES/base.po

```diff
@@ -738,11 +736,23 @@ msgstr "Dodaj: "
 msgid "Value: "
 msgstr "Wartość: "
 
-msgid "Add :"
-msgstr "Dodaj :"
+msgid "You can skip selecting a drive and partitioning and use whatever drive-setup is mounted at /mnt (experimental)"
+msgstr ""
+
+msgid "Select one of the disks or skip and use /mnt as default"
+msgstr ""
```

Comment:
I think OP is Russian, and is only fixing the syntax/format of the PL file to not contain breaking changes.
Meaning it will be hard to do a translation but is able to fix the format issue.

Reply:
> I think OP is Russian, and is only fixing the syntax/format of the PL file to not contain breaking changes.
> Meaning it will be hard to do a translation but is able to fix the format issue.

That's right :)